uo_sys_framework/BriefingModule/cfg3DEN/Briefing/MissionNotesMenu.hpp

```diff
@@ -1,4 +1,4 @@
-class EGVAR(Briefing,MissionNotes) {
+class EGVAR(Briefing,Notes) {
```

Comment:
you've combined two fixes that conflict. All of these vars need to be `(Briefing,Notes)` or `(Briefing,MissionNotes)`

Reply:
whoops, good catch